### ISSUE TITLE
Add optional preprocessing pipelines

### ISSUE THREAD

**Issue by hrshdhgd** [INCATools/ontology-access-kit#360]
Reference: https://github.com/monarch-initiative/mondo-ingest/issues/112#issuecomment-1329314132

- roman numerals: A function call to convert arabic to Roman numerals (or maybe vice versa? )
- stop word removal: similar to `--exclude-tokens` implemented in [PR here](https://github.com/INCATools/ontology-access-kit/pull/353)
- plural: I think the `Lemmatization` pipeline already take scare of this?

**Comment by cmungall** [comment]
roman numerals can be covered by synonymizer?

**Comment by matentzn** [comment]
Yes, but not easily: you will need one rule per numeral? Or are you thinking an extension to synonymiser?
